docs/guides/building/arm-cross-compile.md

```diff
@@ -0,0 +1,162 @@
+# Cross-Compiling Tauri Applications for ARM-based Devices
+
+This guide explains how to cross-compile your Tauri application for ARM-based devices, such as the Raspberry Pi. Compiling directly on the device may not be feasible due to limited RAM, so we'll explore two methods: manual compilation using Linux or WSL on Windows and automatic cross-compilation using a GitHub Action.
+
+## Manual Compilation
+
+Manual compilation is suitable when you don't need to compile your application frequently and prefer a one-time setup. Follow the steps below:
+
+### Prerequisites
+
+:::warning Important Note
+
+Ubuntu 20.04 WSL has issues with webkit dependencies. It's recommended to use Debian instead.
+
+:::
+
+- For Windows, install Debian on WSL using the [wsl setup guide].
+- On Linux, ensure your machine has the same GLIBC version as the target device. Check the version using: `ldd --version`.
```

Comment:
```suggestion
- On Linux, ensure your build machine has an older GLIBC version than the target device. Check the version using: `ldd --version`.
```

It must be the same version or an older version, having troubles making it sound like correct english so i just wrote "older". If you can come up with something better feel free to change it.

Reply:
I unfortunately couldn't find a better word to describe it. I came up with:
On Linux, the build machine requires GLIBC version equal to or older than the target device.

Which is basically what you suggested